students/ryba_paula/lesson_05_lists/age_calculator.py

```diff
@@ -0,0 +1,23 @@
+def age_calculator(a):
+    adults = [i for i in a if i >= 18]
+    children = [i for i in a if i < 18]
+
+    if len(adults) > 0:
```

Comment:
Można w tej sytuacji również użyć:  `if adults:`, który równoważny jest do: `if adults is not None and len(adults) > 0:`

Reply:
dzięki, nie wiedziałam o tym!